core/util/src/main/scala/net/liftweb/util/CombParserHelpers.scala

```diff
@@ -120,13 +120,13 @@ trait CombParserHelpers {
    * @return a parser discarding end of lines
    */
   def EOL: Parser[Unit] = (accept("\n\r") | accept("\r\n") | '\r' |
-                           '\n' | '\032' ) ^^^ ()
+                           '\n' | '\u001a' ) ^^^ ()
 
   def notEOL: Parser[Elem] = (not(EOL) ~> anyChar)
 
-  def notEOF: Parser[Elem] = (not(accept('\032')) ~> anyChar)
+  def notEOF: Parser[Elem] = (not(accept('\u001a')) ~> anyChar)
 
-  def anyChar: Parser[Elem] = elem("Any Char", c => c != '\032')
+  def anyChar: Parser[Elem] = elem("Any Char", c => c != '\u001a')
```

Comment:
Not necessarily for this PR, but it's probably worth sticking this in a named constant, maybe?